apps/core/templates/components/footer.html

```diff
@@ -1,51 +1,26 @@
+{% load wagtailcore_tags %}
 
 <div class="footer">
     <div class="footer__container">
         <div class="footer__link-blocks">
+          {% for footer_item in footer.footer_items.all %}
             <div class="link-block">
```

Comment:
I don't see the `target_link` being used, can we turn this into an `<a>` element with the `target_link` as the `href`?